common/appointment/create.ts

```diff
@@ -35,13 +36,41 @@ export abstract class AppointmentCreateGroupInput extends AppointmentCreateInput
     appointmentType: 'group';
 }
 
+const isAppointmentOneWeekLater = (datestring: Date) => {
+    const now = moment.now();
+    const start = moment(datestring);
+    const diffDays = start.diff(now, 'days');
+    if (diffDays >= 6) {
+        return true;
+    }
+    return false;
+};
+
+const isAppointmentFiveMinutesLater = (datestring: Date) => {
+    const now = moment.now();
+    const start = moment(datestring);
+    const same = start.isSame(now, 'date');
+    const after = start.isAfter(now);
+    if (same) {
+        const diff = start.diff(now, 'minutes');
+        if (diff >= 5) {
+            return true;
+        }
```

Comment:
The moment the request is hitting the api it might be only 4 minutes?
I'm not sure how to handle that correctly, but going with 5 minutes fails I guess, if we're sending the request to a remote server.